### ISSUE TITLE
The Unforgiven boss

### ISSUE THREAD

**Issue by Szjerdene** [dalaranwow/dalaran-wow#4280]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Boss The Unforgiven is not spawning in Stratholme. He should spawn when you walk through rat gate.
**Expected behaviour**: Tell us what should happen instead
Nothing is happening. That's the problem
**Steps to reproduce the problem**:

1.  Just spawn him there - or make an "trap" event. It's easy. He should ambush (spawn) those who run through the rat gate. It's near market lane postbox
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
npc=10516
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/
https://www.youtube.com/watch?v=dBbfXVhU790&t=375s  --> 6:07 ---That's how should it be!!!!
**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Ref: https://github.com/dalaranwow/dalaran-wow/issues/1817

**Comment by Rushor** [comment]
DB/Creature: Stratholme - Add Spawnevent for Boss 'The Unforgiven' in Stratholme 

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4280
* updates: https://github.com/dalaranwow/dalaran-wow/issues/1817
* commit: 451fa59e53f8c3318eff03841166e6f4ac097b0c

![GitHub Logo](http://i.imgur.com/5Sc7nS6.png)
![GitHub Logo](http://i.imgur.com/h5p0ONm.png)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
